integration-tests/api/__tests__/admin/draft-order.js

```diff
@@ -71,6 +71,88 @@ describe("/admin/draft-orders", () => {
       expect(response.status).toEqual(200)
     })
 
+    it("creates a draft order with a billing address that is an AddressPayload and a shipping address that is an ID", async () => {
```

Comment:
suggestion(if-minor): We could add a couple of assertions on the actual shipping address + billing address in both of these tests to be 100% sure we've corrected the issue and to avoid future bugs related to it.

Reply:
They aren't returned as part of the request (not part of default relations), but I can add a fetch afterwards check it against that.

Reply:
Alright, but maybe just asserting on both ids would suffice then?

Reply:
Both the `shipping_address_id` and `billing_address_id` for both tests